Libs/Core/ctkAbstractJob.h

```diff
@@ -177,6 +178,16 @@ class CTK_CORE_EXPORT ctkAbstractJob : public QObject
   /// \sa ctkJobDetail
   Q_INVOKABLE virtual QVariant toVariant();
 
+  /// Free used resources from job after worker is done
+  Q_INVOKABLE virtual void freeUsedResources() = 0;
+
+  ///@{
+  /// Destroy job pointer after worker is done
```

Comment:
Do you mean destroying the job object? A pointer can be set to 0, or if it is a smart pointer then the pointer could be destroyed, but I don't think this is what is meant here.